frontend/packages/ceph-storage-plugin/src/components/modals/storage-pool-modal/storage-pool-modal.tsx

```diff
@@ -121,7 +122,25 @@ export const StoragePoolModal = withHandlePromise((props: StoragePoolModalProps)
     }
   }, [isSubmit, newPool, newPoolLoadError, newPoolLoaded, timer]);
 
-  const replicaDropdownItems = _.keys(OCS_DEVICE_REPLICA).map((replica) => {
+  const [storageCluster] = useK8sGet<ListKind<StorageClusterKind>>(
+    OCSServiceModel,
+    null,
+    CEPH_STORAGE_NAMESPACE,
+  );
+
+  const isArbiterCluster: boolean = storageCluster?.items[0]?.spec?.arbiter?.enable;
+
+  React.useEffect(() => {
+    if (isArbiterCluster) {
+      setReplicaSize('4');
+    }
+  }, [isArbiterCluster]);
+
+  const replicaList: string[] = _.keys(OCS_DEVICE_REPLICA).filter(function(replica: string) {
+    return (isArbiterCluster && replica === '4') || (!isArbiterCluster && replica !== '4');
+  });
```

Comment:
It is not mandatory to keep this (isArbiterCluster && replica === '4'), anyway we are hiding the list for the arbiter, But for safety, i am keeping this.